core/src/main/java/io/aiven/klaw/service/TopicSyncControllerService.java

```diff
@@ -136,7 +136,7 @@ public Map<String, Object> getReconTopics(
     topicRequestModelList.forEach(
         topicReq -> {
           topicReq.setEnvironmentName(getEnvDetails(envId).getName());
-          topicReq.setTopicstatus("ON_CLUSTER");
+          topicReq.setRequestStatus(RequestStatus.APPROVED);
         });
```

Comment:
This is a slightly confusing change can you explain ON_CLUSTER->APPROVED?

Reply:
I had no choice here to mention that this topic exists on cluster. We don't need this field on FE however.
Has no impact for us. Hence removed it.